.github/workflows/build-backend.yml

```diff
@@ -1,11 +1,8 @@
-name: Build backend
+name: Lint, check and build backend
```

Comment:
Do we always need to update the name once we add a new step. this does not fit the file name anymore

Reply:
No, we don't. In fact, we can even set all the names to "coffeecoffeecoffee" and gh actions wouldn't mind. We could clean up the actions structure and everything when we do the first release?